.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ before_script:
   - export DEPLOYMENT_STAGE=$CI_COMMIT_REF_NAME
   - if [ "$DEPLOYMENT_STAGE" == "master" ]; then
   -   export DEPLOYMENT_STAGE=dev
-  - elif [[ "$DEPLOYMENT_STAGE" != "integration" && "$DEPLOYMENT_STAGE" != "staging" ]]; then
+  - elif [[ "$DEPLOYMENT_STAGE" != "integration" && "$DEPLOYMENT_STAGE" != "staging" && "DEPLOYMENT_STAGE" != "prod" ]]; then
```

Comment:
Why do we need reference to prod in non prod gitlab file?

Reply:
Removing